### PR TITLE
Update Time schema to support EarthLocation

### PR DESCRIPTION
**Pull request by drdavella**
The older `v1.0.0` schema for `astropy.Time` did not properly account for the `location` attribute. This PR adds a new `v1.1.0` schema that allows this property to be represented correctly.

### CODE REVIEW

File: schemas/stsci.edu/asdf/time/time-1.1.0.yaml [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+          description: |
+            Specifies the observer location for scales that are
+            sensitive to observer location, currently only `tdb`.  May
+            be specified either with geocentric coordinates (X, Y, Z)
+            with an optional unit or geodetic coordinates:
+              - `long`: longitude in degrees
+              - `lat`: in degrees
+              - `h`: optional height
+
+          anyOf:
```
Comment (nden) [inline schemas/stsci.edu/asdf/time/time-1.1.0.yaml]: Is this `anyOf` necessary here, given that there's only one option. Or am I misreading it?

File: schemas/stsci.edu/asdf/time/time-1.1.0.yaml [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+            - type: object
+              properties:
+                x:
+                  $ref: "../unit/quantity-1.1.0"
+                y:
+                  $ref: "../unit/quantity-1.1.0"
+                z:
+                  $ref: "../unit/quantity-1.1.0"
+              required: [x, y, z]
```
Comment (nden) [inline schemas/stsci.edu/asdf/time/time-1.1.0.yaml]: Are numbers + units not allowed any more? That's fine, I just want to confirm this was the intention.

File: schemas/stsci.edu/asdf/time/time-1.1.0.yaml (hunk 1 above)
Comment (drdavella) [reply schemas/stsci.edu/asdf/time/time-1.1.0.yaml]: Good catch, I think this was just an oversight.

File: schemas/stsci.edu/asdf/time/time-1.1.0.yaml (hunk 2 above)
Comment (drdavella) [reply schemas/stsci.edu/asdf/time/time-1.1.0.yaml]: No, that is no longer allowed, although v1.0.0 of the schema should still be interpreted properly.

**Comment by drdavella** [comment]
@nden that occurred to me as well but I just wanted to do the simple fix first. I'll open an issue in `asdf`.

**Pull request merged**